### ISSUE TITLE
Is it possible to implement Elasticsearch storage adapter

### ISSUE THREAD

**Issue by hanabi1224** [gunthercox/ChatterBot#1513]
which is more scalable to large conversation dataset

**Comment by vkosuri** [comment]
Yes, you have to write a new storage adapter, more information how to write a storage adapter found here https://chatterbot.readthedocs.io/en/stable/storage/create-a-storage-adapter.html
